src/components/DraggableTable/readme.md

```diff
@@ -0,0 +1,51 @@
+```js
+import { useState } from 'react';
+import { arrayMove } from 'react-sortable-hoc';
+
+import { DraggableTable } from './index';
```

Comment:
In readme files use external import to make examples more useful.
It was correct here: https://github.com/tarantool/front-ui-kit/pull/222/files#diff-5f6a3d67c03765a780cb98583b7b7fde6148a1eb44830a868426c08be5dc337fR12.